docker-compose.yml

```diff
@@ -23,37 +23,47 @@ services:
       args:
         ENVIRONMENT: development
     entrypoint: ""
-    command: ["../scripts/watch.sh", "api", "bcda", "start-api"]
+    command: ["bcda", "start-api"]
     env_file:
       - ./shared_files/decrypted/local.env
-    environment:
-      - CI=${CI} # Used to determine if the API is running on a CI process
-    volumes:
-      - .:/go/src/github.com/CMSgov/bcda-app
     ports:
       - "3000:3000"
       - "3001:3001"
     depends_on:
       - ssas
       - db
       - queue
+    develop:
+      watch:
+        - action: rebuild
+          path: ./bcda
+          ignore: ["!**/*.go"]
+        - action: rebuild
+          path: .
+          ignore: ["!go.mod", "!go.sum", "!tools.go"]
+    stop_grace_period: 1s
```

Comment:
As an additional change, whenever we ask the docker containers to stop, the length of time docker waits before force stopping has been reduced from 10s to 1s. I made this a default because the api services are designed to ignore gentle stop commands.